flutter/test/file_system_transport_test.dart

```diff
@@ -70,9 +70,10 @@ void main() {
     );
     await transport.send(envelope);
 
-    final envelopeData = verify(fixture.binding.captureEnvelope(captureAny))
-        .captured
-        .single as Uint8List;
+    final envelopeData =
+        verify(fixture.binding.captureEnvelope(captureAny, false))
```

Comment:
@buenaflor Can you add one new test, which check that the `containsUnhandledException=true` is read from the event and passed to the native call?